content/en/tracing/glossary/_index.md

```diff
@@ -106,6 +107,10 @@ You can use custom instrumentation by embedding tracing code directly into your
 
 To learn more, read [Application Instrumentation][26].
 
+## Baggage
+
+Baggage allows you to propagate key-value pairs (aka baggage items), across service boundaries in a distributed system. Unlike trace context, which focuses on trace identifiers, baggage allows for the transmission of business data and other contextual information alongside traces.  
```

Comment:
Imo we should have links here to similar items so users can build their own understanding.

ie. `trace context` should link to [this](https://docs.datadoghq.com/tracing/trace_collection/trace_context_propagation/)